Kava-Labs/blockscout
# Use docker-compose to run Phoenix app

Currently unable to test the docker-compose full build due to local system constraints.

Should be able to test with:
1. Start `evm` chain using kvtool
2. cd to `/docker` directory
3. run `chmod +x entrypoint.sh`
4. run `make start`

Once it's built and running we'll need to confirm that `ETHEREUM_JSONRPC_WS_URL:ws://host.docker.internal:2000` matches up with the kava `evm` chain's websocket.
